### ISSUE TITLE
Height should take into account that notifications differ in height

### ISSUE THREAD

**Issue by Markus00000** [dunst-project/dunst#82]
If I am not mistaken, setting height to 10 is intended to allow dunst grow to 10 lines.

Now that there can be a frame, padding, a larger line height and lines that wrap, it is hard to set height to a sensible value.

About 8 to 30 notifications will fit my screen depending on their content. What value would I set height to?

Therefore I propose to measure height in pixels (as is width, x, y, padding and frame).

Setting height to a number of pixels would cause any notification that does not fit into the remaining pixels to be queued until there are enough free pixels. Or expressed more simply:

```
if window_height_max - window.height >= notification.height:
    notification.show()
else:
    notification.queue()
```

**Comment by WhyNotHugo** [comment]
I believe this can be closed with #855 merged?

**Comment by fwSmit** [comment]
> I believe this can be closed with #855 merged?

Yes, thank you :+1:
